python/lsst/ip/diffim/subtractImages.py

```diff
@@ -642,7 +655,7 @@ def _subtractImages(science, template, backgroundModel=None):
     return difference
 
 
-def _shapeTest(psf1, psf2):
+def _shapeTest(exp1, exp2, **kwargs):
```

Comment:
Use `evaluateMeanPsfFwhm` and `fwhmExposureGrid` as keywords here, not hidden in `**kwargs`!